src/cascadia/TerminalSettingsEditor/EditColorScheme.cpp

```diff
@@ -45,7 +45,7 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         if (_ViewModel.IsInBoxScheme())
         {
             // load disclaimer for in-box profiles
-            disclaimer = RS_(L"ColorScheme_RenameDisclaimerInBox");
+            disclaimer = RS_(L"ColorScheme_DeleteButtonDisclaimerInBox/Text");
```

Comment:
Ah, I see. When I downloaded this branch, I honestly didn't even read this. I was more like "where is the delete button"? Yeah, I think this should line should be reverted and the delete button (and disclaimer) should be handled with `IsEnabled` and display the disclaimer next to the button.

Reply:
resolving this to continue this discussion in the other [comment](https://github.com/microsoft/terminal/pull/14470#discussion_r1037588939)